src/containers/Header/breadcrumbs.tsx

```diff
@@ -55,167 +61,141 @@ const getClusterBreadcrumbs = (
 
-    // Cluster system tablets don't have tenantName
-    if (tenantName) {
-        breadcrumbs = getTenantBreadcrumbs(options, newQuery);
-    } else {
-        breadcrumbs = getClusterBreadcrumbs(options, query);
-    }
+    const breadcrumbs = tenantName
+        ? getTenantBreadcrumbs(options, {...query, ...tenantQuery})
+        : getClusterBreadcrumbs(options, query);
 
     const link = createHref(routes.tabletsFilters, undefined, {
         ...query,
         nodeIds,
         path: tenantName,
     });
 
-    breadcrumbs.push({text: headerKeyset('breadcrumbs.tablets'), link});
+    const lastItem = {text: headerKeyset('breadcrumbs.tablets'), link};
+    breadcrumbs.push(lastItem);
 
     return breadcrumbs;
 };
 
-const getTabletBreadcrubms = (
-    options: TabletBreadcrumbsOptions,
-    query = {},
-): RawBreadcrumbItem[] => {
+const getTabletBreadcrumbs: GetBreadcrumbs<TabletBreadcrumbsOptions> = (options, query = {}) => {
     const {tabletId, tabletType} = options;
 
-    const breadcrumbs = getTabletsBreadcrubms(options, query);
+    const breadcrumbs = getTabletsBreadcrumbs(options, query);
 
-    breadcrumbs.push({
+    const lastItem = {
         text: tabletId || headerKeyset('breadcrumbs.tablet'),
         icon: <TabletIcon text={getTabletLabel(tabletType)} />,
-    });
+    };
+
+    breadcrumbs.push(lastItem);
 
     return breadcrumbs;
 };
 
+const mapPageToGetter = {
+    cluster: getClusterBreadcrumbs,
+    node: getNodeBreadcrumbs,
+    pDisk: getPDiskBreadcrumbs,
+    tablet: getTabletBreadcrumbs,
+    tablets: getTabletsBreadcrumbs,
+    tenant: getTenantBreadcrumbs,
+    vDisk: getVDiskBreadcrumbs,
+} as const;
+
 export const getBreadcrumbs = (
     page: Page,
     options: BreadcrumbsOptions,
     rawBreadcrumbs: RawBreadcrumbItem[] = [],
     query = {},
 ) => {
-    switch (page) {
-        case 'cluster': {
-            return [...rawBreadcrumbs, ...getClusterBreadcrumbs(options, query)];
-        }
-        case 'tenant': {
-            return [...rawBreadcrumbs, ...getTenantBreadcrumbs(options, query)];
-        }
-        case 'node': {
-            return [...rawBreadcrumbs, ...getNodeBreadcrumbs(options, query)];
-        }
-        case 'pDisk': {
-            return [...rawBreadcrumbs, ...getPDiskBreadcrumbs(options, query)];
-        }
-        case 'vDisk': {
-            return [...rawBreadcrumbs, ...getVDiskBreadcrumbs(options, query)];
-        }
-        case 'tablets': {
-            return [...rawBreadcrumbs, ...getTabletsBreadcrubms(options, query)];
-        }
-        case 'tablet': {
-            return [...rawBreadcrumbs, ...getTabletBreadcrubms(options, query)];
-        }
-        default: {
-            return rawBreadcrumbs;
-        }
-    }
+    if (!page) return rawBreadcrumbs;
+
+    const getter = mapPageToGetter[page];
+
+    return [...rawBreadcrumbs, ...getter(options, query)].map((item) => ({
+        ...item,
+        action: () => {},
```

Comment:
let's return here raw breadcrumbs without action (cause you already add action in https://github.com/ydb-platform/ydb-embedded-ui/pull/913/files#diff-aa4640b8027696409b37009a150d396684a06d0dd3d7ec6c4abbeb6a74956480R51

Reply:
I guess this placement is more appropriate but actually not a big deal
Done